src/components/SystemPages/NewReaderControls.vue

```diff
@@ -0,0 +1,377 @@
+<template>
+  <div class="newReaderControls">
+    <template v-if="featureList.includes('pagenumber')">
+      <!-- <span v-if="$localData.settings['devMode']" v-text="newReaderPage" /> -->
+      <div class="newReaderInput">
+        <!-- Settings for adjusting new reader mode -->
+        <template v-if="$isNewReader">
+          <p>New reader mode enabled.</p>
+          <!-- Can't show picker if you're in viz mode. -->
+          <StoryPageLink
+            v-if="isNewReadingPreHS" 
+            :mspaId='$newReaderCurrent'/>
+          <input type="number" size="1" maxlength="6" 
+            v-model="newReaderPageInput"
+            v-else
+            @keydown.enter="changeNewReader()"
+            :class="{
+              invalid: !isValidPageSet, 
+              empty: !newReaderPageInput || !newReaderPageInput.length, 
+              changed: newReaderPageChanged
+            }" >
+            <span :class="{ urgent: newReaderPageChanged }">
+            <button :disabled="!(isValidPageSet && newReaderPageChanged)" 
+              @click="changeNewReader()" 
+              class="rightBtn"
+              >Adjust</button>
+            </span>
+        </template>
+        <template v-else>
+          <p>New reader mode disabled.</p>
+          <input type="number" size="1" maxlength="6" 
+            v-model="newReaderPageInput"
+            @keydown.enter="setNewReader()"
+            :class="{
+              invalid: !isValidPageSet, 
+              empty: !newReaderPageInput || !newReaderPageInput.length, 
+              changed: newReaderPageChanged
+            }" >
+          <span :class="{ urgent: newReaderPageChanged }">
```

Comment:
This doesn't trigger for the Activate button, not sure why

Reply:
Bug, fixed